apps/web/src/components/SpeakersList/SpeakersList.tsx

```diff
@@ -33,23 +35,18 @@ export default function SpeakersList() {
   const classes = useStyles();
 
   return (
-    <>
+    <div className={classes.speakersListContainer}>
       <div className={classes.header}>{`Speakers (${speakers.length})`}</div>
+      <Typography variant="body1">{`${host} (Host)`}</Typography>
 
-      <div className={classes.speakersListContainer}>
-        <Typography variant="body1">{`${host} (Host)`}</Typography>
-
-        {speakers
-          .filter(speaker => speaker !== host)
-          .map(speaker => (
-            <div className={classes.speakerContainer}>
-              <Typography key={speaker} variant="body1">
-                {speaker}
-              </Typography>
-              {localParticipant?.identity === host && <SpeakerMenu />}
-            </div>
-          ))}
-      </div>
-    </>
+      {speakers
+        .filter(speaker => speaker !== host)
+        .map(speaker => (
+          <div key={speaker} className={classes.speakerContainer}>
+            <Typography variant="body1">{speaker}</Typography>
```

Comment:
What about displaying `speaker + '(You)'` when the speaker here is the local participant?

Reply:
yeah I like it!